### PR TITLE
Update branding to 3.1.6

### PR DESCRIPTION
**Pull request by wtgodbe**

### CODE REVIEW

**Review by ryanbrandenburg — APPROVED** [review]

File: src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/appsettings.json [hunk 1]
```diff
@@ -10,9 +10,9 @@
 ////#endif
   "Logging": {
       "LogLevel": {
-      "Default": "Information",
```
Comment (JunTaoLuo) [inline src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/appsettings.json:13]: Huh what changed here?

**Review by JunTaoLuo — APPROVED** [review]

**Review by dougbu — APPROVED** [review]

File: src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/appsettings.json (hunk 1 above)
Comment (wtgodbe) [reply src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/appsettings.json:13]: Not sure - something weird in the merge, I guess. I can push a commit undoing this

File: eng/Baseline.xml [hunk 2]
```diff
@@ -4,86 +4,86 @@ This file contains a list of all the packages and their versions which were rele
...
+  <Package Id="Microsoft.AspNetCore.Authentication.Twitter" Version="3.1.5" />
+  <Package Id="Microsoft.AspNetCore.Authentication.WsFederation" Version="3.1.5" />
+  <Package Id="Microsoft.AspNetCore.Authorization" Version="3.1.5" />
+  <Package Id="Microsoft.AspNetCore.AzureAppServices.HostingStartup" Version="3.1.5" />
+  <Package Id="Microsoft.AspNetCore.AzureAppServices.SiteExtension" Version="3.1.5" />
+  <Package Id="Microsoft.AspNetCore.AzureAppServicesIntegration" Version="3.1.5" />
   <Package Id="Microsoft.AspNetCore.Blazor" Version="3.2.0-preview1.20073.1" />
   <Package Id="Microsoft.AspNetCore.Blazor.Build" Version="3.2.0-preview1.20073.1" />
   <Package Id="Microsoft.AspNetCore.Blazor.DevServer" Version="3.2.0-preview1.20073.1" />
   <Package Id="Microsoft.AspNetCore.Blazor.HttpClient" Version="3.2.0-preview1.20073.1" />
```
[the remark this answers is not in the document]
Comment (wtgodbe) [reply eng/Baseline.xml:31]: The version you mentioned is newer than the preview1 version currently in the file

File: eng/Baseline.xml [hunk 3]
```diff
@@ -4,86 +4,86 @@ This file contains a list of all the packages and their versions which were rele
...
   <Package Id="Microsoft.AspNetCore.Blazor.Server" Version="3.2.0-preview1.20073.1" />
   <Package Id="Microsoft.AspNetCore.Blazor.Templates" Version="3.2.0-preview1.20073.1" />
```
[the remark this answers is not in the document]
Comment (wtgodbe) [reply eng/Baseline.xml:33]: I'll remove these - @pranavkm is adding the new ones in https://github.com/dotnet/aspnetcore/pull/21921/files

File: src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/appsettings.json (hunk 1 above)
Comment (wtgodbe) [reply src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/appsettings.json:13]: Replacing the file locally doesn't change anything in git, but I think this is a no-op anyways

**Pull request merged**